test/manipulation.test.js

```diff
@@ -544,31 +544,72 @@ describe('manipulation', function() {
         });
     });
 
-    it('should ignore unknown attributes when strict: true', function(done) {
-      // Using {foo: 'bar'} only causes dependent test failures due to the
-      // stripping of object properties when in strict mode (ie. {foo: 'bar'}
-      // changes to '{}' and breaks other tests
```

Comment:
moved it under `it('should return error on unknown attributes when strict: true',`